PriceQuoteIos/Models/Salesman.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Salesman.swift
+//  PriceQuoteIos
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+struct Salesman {
+    let name: String
+    let status: SalesmanStatus
+    let phone: Phone
+    
+    func format() -> [String: Any] {
+        return [
+            "salesman": [
+                "name": name, 
+                "status": status.rawValue,
+                "phone": [
+                    ["number": phone.number,
+                    "whatsapp": phone.isWhatsappActive]
+                ]
+            ]
+        ]
+    }
+}
+
+struct Phone {
+    let number: String
+    let isWhatsappActive: Bool
+}
+
+public enum SalesmanStatus: String {
+    case active = "active"
+    case inactive = "inactive"
+}
+
+public class BaseResponse: Mappable {
+    var status: Int?
+    var result: Mappable?
+    
+    convenience required public init?(map: Map) {
+        self.init()
+    }
+    
+    public func mapping(map: Map) {
+        status <- map["status"]
+        result <- map["data"]
+
+    }
+}
```

Comment:
Same comment as above, but you wouldn't even need the mapping since `Codable` takes care of that for you. But, I don't remember exactly if codable works fine with Alamofire!

Reply:
I removed Alamofire :)